### DISCUSSION TITLE
Bought Copilot for Organization, still can't use it

### DISCUSSION THREAD

**Discussion by callebtc** [community/community#51838] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Our org bought Copilot and activated it. My Github account shows that I am eligable. 

I am still getting these errors in VSCode. I tried signing out and in into Github on VSCode. Nothing seems to work.

```
[INFO] [auth] [2023-04-03T12:21:45.757Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2023-04-03T12:21:45.758Z] GitHub Copilot could not connect to server. Extension activation failed: "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended."
```

**Comment by Akash1134** [comment]
Thanks for getting in touch @callebtc 

In VSCode, go to the accounts menu (a circle with a head shape inside it) on the lower left-hand sidebar. Log out of any GitHub accounts that are there, and then log in making sure that you are logging into the correct GitHub account using which your Org signed you up for Copilot.

Please let me know if it helped 😊
